### ISSUE TITLE
CMake build is still making some invalid libs

### ISSUE THREAD

**Issue by Ryp** [psp2dev/psp2sdk#55]
When compiling vitahelloworld:
```
main.c:(.text+0x68c): undefined reference to `sceKernelGetProcessTimeWide'
draw.o: In function `alloc_gpu_mem':
draw.c:(.text+0x80): undefined reference to `sceKernelAllocMemBlock'
draw.c:(.text+0xbc): undefined reference to `sceKernelGetMemBlockBase'
draw.o: In function `swap_buffers':
draw.c:(.text+0x47c): undefined reference to `sceDisplaySetFrameBuf'
```

**Comment by 173210** [comment]
Those libraries include stubs for several modules.

**Comment by Ryp** [comment]
Fixed when not mixing NIDS between modules. Will commit a fix tomorrow

**Comment by Ryp** [comment]
Fixed in commit 0b726c5c04999ba854a990681c4f46962d417c92
